docs/assertions/assertions.mdx

```diff
@@ -14,8 +16,30 @@ $true | Should -Be $true
 $true | Should -Not -Be $false
 ```
 
+### Because
+
+Lets you to add an optional message to the default test error message to tell the user why the test should pass.
+
+```powershell
+$actual = "Actual value"
```

Comment:
Maybe reuse the Get-Planet example with `-Name M*` which should return 'Mercur, Mars' -Because "those are the only two planets starting with M*'?

Reply:
Updated example